### PR TITLE
New allocation request

### PR DESCRIPTION
**Pull request by dabasov**

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | alloc-blob  |
| gosdk  | alloc_blobber_stable  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | alloc-blob  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | new_alloc  |
| zwalletcli  | staging  |
| 0chain  | alloc_blob_stable  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | alloc-blob  |
| gosdk  | alloc_blobber_stable  |
| zboxcli  | new_alloc  |
| zwalletcli  | staging  |
| 0chain  | alloc_blob_stable  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

File: tests/cli_tests/config/zbox_config.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-block_worker: https://dev-2.devnet-0chain.net/dns
```
Comment (mohsenno1) [inline tests/cli_tests/config/zbox_config.yaml:1]: I think this has to be returned back to https://dev.0chain.net/

**Review by mohsenno1 — APPROVED** [review]

File: tests/cli_tests/zwalletcli_send_and_balance_test.go [hunk 2]
```diff
@@ -307,14 +307,18 @@ func sendZCN(t *testing.T, cliConfigFilename, toClientID, tokens, desc string, r
...
+func sendTokensFromWallet(t *testing.T, cliConfigFilename, toClientID string, tokens float64, desc string, fee float64, wallet string) ([]string, error) {
 	t.Logf("Sending ZCN...")
 	cmd := fmt.Sprintf(`./zwallet send --silent --tokens %v --desc %q --to_client_id %s `, tokens, desc, toClientID)
 
 	if fee > 0 {
 		cmd += fmt.Sprintf(" --fee %v ", fee)
 	}
 
-	cmd += fmt.Sprintf(" --wallet %s --configDir ./config --config %s ", escapedTestName(t)+"_wallet.json", cliConfigFilename)
+	cmd += fmt.Sprintf(" --wallet %s --configDir ./config --config %s ", wallet+"_wallet.json", cliConfigFilename)
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zwalletcli_send_and_balance_test.go:321]: Nice catch

**Review by Kishan-Dhakan — APPROVED** [review]
Looks Good to me

**Pull request merged**